### PR TITLE
Csv encoding

### PR DESCRIPTION
**Pull request by alxwest**
I found the issue of the Ace engine not using the correct encoding for the CSV file I was using. 

I've put in the ability to set the Code Page Identifier from the ExcelQueryFactory so it uses the correct encoding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -326,6 +326,13 @@ See [Pooling in the Microsoft Data Access
 Components](https://msdn.microsoft.com/en-us/library/ms810829.aspx#Troubleshooting%20MDAC%20Pooling) for more
 information.
 
+## Encoding
+
+If the file is in a different encoding use the CodePageIdentifer so the setting is passed to the engine.
+
+See [Code Page Identifiers](https://docs.microsoft.com/en-us/windows/desktop/intl/code-page-identifiers) for a complete 
+listing of Code Page Identifiers and their corresponding encoding.
+
```
Comment (mrworkman) [inline README.md]: Please add a specific code example showing users how, and when, to set this new property.

File: README.md [hunk 2]
```diff
@@ -326,6 +326,13 @@ See [Pooling in the Microsoft Data Access
...
 ```c#
 var excel = new ExcelQueryFactory("excelFileName");
 excel.OleDbServices = Query.OleDbServices.AllServicesExceptPoolingAndAutoEnlistment;
```
Comment (mrworkman) [inline README.md]: Your encoding section should really come after this other code example, since it belongs to the previous section.

Comment (alxwest) [reply README.md]: Done 👍

File: README.md (hunk 1 above)
Comment (alxwest) [reply README.md]: Done 👍

**Comment by mrworkman** [comment]
@alxwest merged, thanks!

**Comment by Hect0rr** [comment]
This feature wasn't included in the pre-releaase 2.0.0. Do you have a date for the next stable release?

**Comment by mrworkman** [comment]
No, not really. Unfortunately with a recent move + life, etc... it's been hard to spend any time on this. I will try to make a release soon, taking it out of pre-release as well.

**Pull request merged**